cypress/integration/plugins/dashboards-assistant/conversation_history_spec.js

```diff
@@ -79,8 +79,12 @@ if (Cypress.env('DASHBOARDS_ASSISTANT_ENABLED')) {
         cy.get('textarea[placeholder="Ask me anything..."]').should('exist');
       });
 
-      it('should hide back button in fullscreen mode', () => {
-        cy.get('.llm-chat-flyout button[aria-label="fullScreen"]').click();
+      it('should hide back button in takeover fullscreen mode', () => {
+        //switch to takeover mode for fullscreen
+        cy.get('[id="sidecarModeIcon"]').click();
```

Comment:
Shall we change to `cy.get("#sidecarModeIcon")`?

Reply:
I think both ways are OK.